modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -52,11 +55,16 @@ The following problems may exist when you have a server group composed of multip
 
 * *Salesforce streaming API*: if multiple instances of the same application are deployed, they will fail since the API only supports a single consumer. No failover support in case the instance connected is stopped or crashes
 
+== Prerequisites
 
+* You must have at least two Mule runtimes in a cluster, each of which should run on different physical (or virtual) machines.
```

Comment:
Pls change "should" to something else. Maybe "cluster, where each runs on" or something like that.  We're supposed to avoid "should".